Public/Src/Tools/Execution.Analyzer/Args.cs

```diff
@@ -634,9 +624,6 @@ private static void WriteHelp()
             //writer.WriteLine("");
             //WriteXLGToDBHelp(writer);

```

Comment:
- there will be no analyzers here, right? I think we can remove that `todo`
- uncomment the `WriteXLGToDBHelp` ?

Reply:
Correct, there are no other analyzers. I was going to uncomment out the xllgtodb helper after the "interesting indexing" step

---
In reply to: [316308804](https://github.com/microsoft/BuildXL/pull/775#discussion_r316308804) [](ancestors = 316308804)